api_tests/lib/ethereum.ts

```diff
@@ -193,6 +194,7 @@ export class EthereumWallet {
     }
 
     public async signAndSend(tx: TransactionRequest) {
+        await sleep(1001); // instant seal is a beautiful thing, if we fire transactions too fast, than btsieve can't see the block
```

Comment:
*cough* https://github.com/comit-network/comit-rs/issues/1106 *cough*

Reply:
Oh no, did you catch a cold in Tel Aviv? 🤪

... reopening #1106 now